cpp/installer.cc

```diff
@@ -75,7 +76,7 @@
 
 
 [[noreturn]] void Usage() {
-    ::Usage(std::string("--ub-tools-only|--fulltext-backend|(vufind_system_type [--omit-cronjobs] [--omit-systemctl])\n") +
+    ::Usage(std::string("(--production|--test) --ub-tools-only|--fulltext-backend|(vufind_system_type [--omit-cronjobs] [--omit-systemctl])\n") +
```

Comment:
Please also change all affected Dockerfiles that call install_ubtools.sh (which passes the parameters on to installer)